src/engine/ruleset/integrations/windows/decoders/windows-sysmon.yml

```diff
@@ -91,12 +246,20 @@ normalize:
     - related.user: array_append($user.name)
     - related.ip: array_append($windows.EventData.NewTargetUserName)
 
+  ## Sysmon fields
+    - sysmon.dns.status: $windows.EventData.QueryStatus
+    - sysmon.file.archived: $windows.EventData.Archived
+    - sysmon.file.is_executable: $windows.EventData.IsExecutable
+
+ - check: $file.code_signature.status == Valid
+   map:
+     - file.code_signature.valid: True
 
- - check: $event.code=="255"
+ - check: $event.code == 255
```

Comment:
Event.code is a string not a number, it must be enclosed in single quotes, otherwise this check will always fail.